microsoft-365/security/defender-endpoint/linux-exclusions.md

```diff
@@ -36,7 +36,7 @@ ms.date: 12/18/2020
 This article provides information on how to define exclusions that apply to on-demand scans, and real-time protection and monitoring.
 
 > [!IMPORTANT]
-> The exclusions described in this article don't apply to other Defender for Endpoint on Linux capabilities, including endpoint detection and response (EDR). Files that you exclude using the methods described in this article can still trigger EDR alerts and other detections. For EDR exclusions, [contact support](/microsoft-365/admin/get-help-support).
+> The exclusions described in this article don't apply at the level of other Defender for Endpoint on Linux capabilities, including endpoint detection and response (EDR). However, files that you exclude using the methods described in this article can still trigger EDR alerts and other detections. For EDR exclusions, [contact support](/microsoft-365/admin/get-help-support).
```

Comment:
Maybe we can say "... don't apply at the **scope** of other ..."?

Reply:
Kept documentation for Antivirus and Global Exclusions separate.